hr_public_holidays/models/hr_public_holidays.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding:utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>.
+#    Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+#    Author Nikolina Todorova <nikolina.todorova@initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, _
+from datetime import date, datetime
+from openerp.exceptions import ValidationError
+
+
+class HrPublicHolidays(models.Model):
+    _name = 'hr.holidays.public'
+    _description = 'Public Holidays'
+    _rec_name = 'year'
```

Comment:
OK, what I mean is that we need a more complete name, not only the year, but possibly the states between brackets. Instead of using `name_get`, you can use `display_name` computed field, but the principle is to complete the shown info